### DISCUSSION TITLE
タスクビューとスタートボタンが使用できない。

### DISCUSSION THREAD

**Discussion by r-rikka** [creeper-0910/ExplorerPatcher-jp#39] [General]
Win11 Pro 22H2 22621.2215 にて22621.1992.56.3を使用時
Win11スタイルのスタートボタンが機能しません。Win10スタイルでは機能します。
また、タスクビューボタンをクリックしたり、マウスパッドで3本指を上フリックしたときにタスクビューが使用できません。
PC起動後にタスクビューボタンをクリックするかマウスパッドで3本指を上フリックすると、マウスパッドの3本指ジェスチャーが全て使用できなくなります。
フォーク元にも報告しています。

**Comment by r-rikka** [comment]
フォーク元から返事がありました。この問題は大量の報告があるようです。
開発者より以下のツリーに統合されました。https://github.com/valinet/ExplorerPatcher/issues/1687
現状では直るまでの間は代替ツールの使用が推奨されています（EPと同時使用できるみたいです）。
あまり複数の同系統ソフトを入れたくないので、少し不便ですが直るまで待つことにします。

**Comment by creeper-0910** [comment]
READMEを更新します。
このissueはこの問題に関する情報交換の場としてDiscussionに移動します。
ExplorerPatcher-jpの更新と、ご報告ありがとうございます！

**Comment by creeper-0910** [comment]
https://github.com/creeper-0910/ExplorerPatcher-jp/commit/4e10f9427c5bcf4cb88a29bbd5e5b768cdde497c
遅くなってしまい申し訳ございません。今回のコミットで現在の問題について追記しました。
その他追記が必要な問題がありましたらこちらのスレッドにお願いします。

**Comment by Amrsatrio** [comment]
フォーク元の新しいメンテナです。最新のバージョンでは、私がこのバグおよび多くの他の大切なバグを修正しました。ご迷惑をかけて申し訳ございません。

もし大切なバグまたはクラッシュがまだあったら、教えてください。

**Reply by creeper-0910** [reply]
了解致しました、ありがとうございます!

**Reply by creeper-0910** [reply]
バグかどうかはわかりませんが、昔からこのような問題があります。

https://github.com/creeper-0910/ExplorerPatcher-jp/issues/17

**Comment by r-rikka** [comment]
日本語版EP 22621.2361.58.1
OSビルド Win11 Pro 22621.2215
にてタスクビューまわりのバグが直っていることを確認しました。
Win11スタイルのスタートボタンも正常に動作します。

OSは念のためアップデートを延期して使用しています。
最新のOSビルド、またはWin10で正常に動作するかは未確認です。

とりあえず快適に使えるように戻ったので嬉しいです。
機能が多いため日本語訳も大変だと思います、ありがとうございます。
